examples/draft-0-10-0/playground/src/App.css

```diff
@@ -1,28 +1,142 @@
+:root {
+  --nav-main-height: 50px;
+  --main-background: #f7f7f7;
+}
+
 .App {
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.playground-main select,
+.playground-main button {
+  color: #fff;
+  font-size: 12px;
+  font-weight: bold;
+  border: none;
+  padding: 0px 10px;
+  height: 24px;
+  border-radius: 4px;
+}
+
+.playground-main select {
+  background: #843131;
+}
+
+.playground-main button {
+  background: #3B3738;
+  margin-left: 10px;
+}
+
+.playground-main button:hover {
+  background: #843131;
+}
+
+.playground-main select:hover,
+.playground-main button:hover {
+  cursor: pointer;
+}
+
+.nav-main {
+  *zoom: 1;
+  background: #3B3738;
+  color: #fafafa;
+  position: fixed;
+  top: 0;
+  height: var(--nav-main-height);
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.5);
+  width: 100%;
+  z-index: 100;
+  padding: 0 15px;
+}
+
```

Comment:
This styles comes from docussaurus website theme, so maybe @JoelMarcey  have more info around it. I have just coppied it out so that our example playground have similar styles to our website. Eventually we could move the playground to the website itself and delete all this extra styles

Reply:
Hi @mitermayer @flarnie -- we don't have those two exact lines in https://github.com/facebook/Docusaurus/blame/master/lib/static/css/main.css, so I am not quite sure where that was taken from. 

I searched from `z-index: 100;` and most were added in the initial commit and I don't have a huge amount of historical context as to why.

@mitermayer Did you get those two lines from your main.css file somewhere -- maybe if I knew that I could cross-check against the default `main.css` Docusaurus file.

Reply:
So the live Draft.js docs are not on Docusaurus yet - I think we could use some help getting the Docusaurus PR merged. Not sure what the final steps are there. https://github.com/facebook/draft-js/pull/1607

So @mitermayer if you took it from the current website styles, that's fine, probably a clearfix hack.

Reply:
@flarnie I am definitely willing to help you get that merged. 😄 I am going to publish a new version very soon (maybe tonight), and maybe we can work off that?